app/controllers/ActorController.scala

```diff
@@ -5,15 +5,15 @@ import play.api.libs.streams.ActorFlow
 import javax.inject.Inject
 import akka.actor.{ActorSystem, Props}
 import akka.stream.Materializer
-import actor.{Actor, WebSocketActor}
+import actor.{Actor, PublicRoomActor}
 
 class ActorController @Inject()(cc:ControllerComponents)
                                (implicit system: ActorSystem, mat: Materializer) extends AbstractController(cc) {
 
   def socket: WebSocket = WebSocket.accept[String, String] { request =>
     ActorFlow.actorRef { ref =>
-      val act:Props = WebSocketActor.props(ref)
-      Actor.actorList += ref
+      val act:Props = PublicRoomActor.props(ref)
+//      Actor.publicActorList += ref
       act
```

Comment:
What you did is connect WebSocket to `PublicRoomActor` actor you need to connect to `UserActor`
So you need to return `Props` of `UserActor`

Reply:
Ok, when someone connect through socket you need to send message and you did not do that yet 
1. Send message to `PublicRoomActor`
2. return `UserActor` to connect with socket

Reply:
```scala
ActorFlow.actorRef { ref =>
  ChatSystem.lounge ! ref
  UserActor.props
}
```